.pre-commit-config.yaml

```
@@ -28,6 +28,12 @@ repos:
       - id: isort
         args: [--settings-path, ./pyproject.toml]
 
+  - repo: https://github.com/pre-commit/mirrors-yapf
+    rev: v0.30.0
+    hooks:
+      - id: yapf
+        args: [--parallel]
```

doesn't it also need -i and -r?

---

yes, needs `-i`. Not `-r` because pre-commits applies each hook to the appropriate files already

```suggestion
        args: [--parallel, --in-place]
```

---

The nice one is `-rip`